elements/bulbs-reading-list/lib/reading-list-article.js

```diff
@@ -106,8 +106,39 @@ export default class ReadingListArticle {
     this.element.dataset.loadStatus = 'loading';
   }
 
+  setupTaboola () {
+    let taboola = window._taboola || [];
```

Comment:
https://trello-attachments.s3.amazonaws.com/57696846132a1ca9bbebce86/583ef4962b8a783aaed10b08/2d80deb739788ff675db60bb3b46a0b6/implementation.pdf in case you're curious, here are their implementation instructions.